app/scripts/directives/datepicker.js

```diff
@@ -27,35 +27,36 @@ angular
             );
           });
         };
+        $scope.getDateOptions = function (model) {
+          let initialDate = model
```

Comment:
Are you able to do `$scope.localModel` here, or do we need to pass in modal?

Reply:
i just changed that